server/service/integration_mdm_test.go

```diff
@@ -2141,7 +2142,7 @@ func (s *integrationMDMTestSuite) TestMDMAppleUnenroll() {
 	require.Equal(t, "", hostResp.Host.MDM.Name)
 }
 
-func (s *integrationMDMTestSuite) TestMDMAppleGetEncryptionKey() {
+func (s *integrationMDMTestSuite) TestAppleMDMGetEncryptionKey() {
```

Comment:
Lol well I removed the `Apple` part initially to reuse the same test for both Apple and Windows but it was already big and complex, decided to use a distinct test method for Windows instead. Didn't put the Apple back at the right place :D